galaxy.yml

```diff
@@ -9,7 +9,7 @@ namespace: pulp
 name: pulp_installer
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 3.3.1
+version: 3.5.0.dev
```

Comment:
I have yet to to rework the main installer docs, but we need to update the README.md as well.